### PR TITLE
Refine Kubernetes scalability SLOs page

### PR DESCRIPTION
**Pull request by wojtek-t**
1. Update the section how we define Kubernetes scalability and what we require from our SLIs/SLOs. This is mostly based on the Kubecon presentation I gave:
https://www.youtube.com/watch?v=7j76n21Pn-4&feature=youtu.be

2. Remove "burst SLIs/SLOs" - they were WIP anyway, required a bunch of work and there are no plans in any near/medium term future to proceed with them.

3. Extend description of our requirements/prerequisites for SLOs to be satisfied.

/assign @mm4tt @shyamjvs 
@oxdrr @krzysied @mborsz @kubernetes/sig-scalability-pr-reviews

### CODE REVIEW

File: sig-scalability/slos/slos.md [hunk 1]
```diff
@@ -9,104 +9,115 @@ you would expect to have some guarantees in those areas.
...
+We are also introducing two more prerequisites which has to be met to ensure that
+SLOs can be satisfied:
 
 ```
-Prerequisite: Kubernetes cluster is available and serving.
+Prerequisites:
+   1. Kubernetes cluster is available and serving.
+   2. Cluster churn is <= 20, where churn is defined as:
+     churn = $(Pod spec creations/updates/deletions) + #(user originated requests) in a given second
 ```
```
Comment (mm4tt) [inline sig-scalability/slos/slos.md]: s/$/#/?

Comment (wojtek-t) [reply sig-scalability/slos/slos.md]: Done.

**Comment by wojtek-t** [comment]
@mm4tt - PTAL

File: sig-scalability/slos/slos.md [hunk 2]
```diff
@@ -9,104 +9,115 @@ you would expect to have some guarantees in those areas.
...
 
 The currently existing SLIs/SLOs are enough to guarantee that cluster isn't
-completely dead. However, they are not enough to satisfy user's needs in most
-of the cases.
+completely dead. However, they are not meeting user expectations in many areas of
+the system and we are actively working on extending their coverage.
 
-We are looking into extending the set of SLIs/SLOs to cover more parts of
-Kubernetes.
+We are also introducing two more prerequisites which has to be met to ensure that
```
Comment (mm4tt) [inline sig-scalability/slos/slos.md]: NIT: s/has/have/

Comment (mm4tt) [reply sig-scalability/slos/slos.md]: I'll fix it :)

**Pull request merged**